source/fundamentals/crud/write-read-pref.txt

```diff
@@ -60,56 +60,110 @@ write concern:
        |
-       | **Parameter**: ``string``
-       
+       | **Parameter**: none
+
+   * - ``W1()``
+     - | The client requests acknowledgement that write operations have
+         been written to memory on one node, such as the standalone mongod or
+         the primary in a replica set. For more
+         information, see the :rapid:`Write Concern specification for w: 1
+         </reference/write-concern/#mongodb-writeconcern-writeconcern.-number->`.
+       |
+       | **Parameter**: none
+
+If you require a more specialized write concern, you can define a custom
+``WriteConcern`` struct literal. You can set the following fields in a
+``WriteConcern`` struct:
+
+.. list-table::
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Field
+     - Description
+
+   * - ``W``
+     - | Specifies the number of ``mongod`` instances or tagged members
+         that write operations must propagate to for acknowledgement. Common values include
+         ``1``, ``0``, and ``"majority"``.
+       |
+       | **Type**: ``string`` or ``int``
+
+   * - ``Journal``
+     - | Specifies if write operations need to be written to the on-disk
+         journal for acknowledgement.
+       |
+       | **Type**: ``bool``
+
+   * - ``WTimeout``
+     - | Specifies a time limit for the write concern. This setting is
+         applicable only for ``W`` values greater than 1. Specifying this
+         setting and specifying a :ref:`Timeout
+         <golang-timeout-setting>` on the client at the same time
+         results in undefined behavior. To learn more, see the
+         :manual:`Write Concern specification for wtimeout </reference/write-concern/#wtimeout>`.
+       |
+       | **Type**: ``time.Duration``
+
 .. tip::
 
    You can alternatively specify a write concern in your connection
-   string. See the :manual:`Server Manual entry on Write Concern Options
+   string. See the :manual:`Server manual entry on Write Concern Options
    </reference/connection-string/#write-concern-options>` for more information.
 
 Example
 ~~~~~~~
 
-The following code shows how you can specify a write concern to request
-acknowledgement from two replica set members. The code then creates a ``Client``
-with this option.
+The following code shows how you can specify different write concerns
+at the client and collection level. The *client-level* write concern
+requests acknowledgement from two replica set members and sets journaling to
+``false``. The *collection-level* write concern requests
+acknowledgement from the majority of replica set members.
 
 .. code-block:: go
-   :emphasize-lines: 2-3
+   :emphasize-lines: 2-6,10-11
    
    uri := "mongodb://<hostname>:<port>"
-   wc := writeconcern.W(2)
-   opts := options.Client().ApplyURI(uri).SetWriteConcern(writeconcern.New(wc))
-   client, err := mongo.Connect(context.TODO(), opts)
+   cliWC := &writeconcern.WriteConcern{
+       W: 2,
+       Journal: false,
```

Comment:
`Journal` is a `*bool`, so you can't assign a bool literal to it. It needs to be updated to something like:
```go
journal := false
cliWC := &writeconcern.WriteConcern{
    W: 2,
    Journal: &journal,
}
```